x-pack/solutions/security/plugins/security_solution/public/data_view_picker/redux/listeners/readme.md

```diff
@@ -0,0 +1,3 @@
+# Side effects for data view picker
+
+We are currently using redux toolkit listener middleware to implement side effects logic in Data View Picker for Security Solution plugin.
```

Comment:
We should flesh this out more to give a high level of the data flow and listeners. It could even live at the top of the `data_view_picker` folder here